indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -248,6 +252,12 @@
             $checkbox.prop('checked', !$checkbox.prop('checked')).trigger('change');
             toggleResetBtn();
         });
+
+        var $eventParticipantsList = $('#event-participants-list');
+
+        $eventParticipantsList.on('indico:htmlUpdated', function () {
```

Comment:
This could be simplified like this (we don't put a space between `function` and `()` btw):

    $('#event-participants-list').on('indico:htmlUpdated', initTooltip);